src/lights/PointLight.js

```diff
@@ -10,6 +10,7 @@ THREE.PointLight = function ( color, intensity, distance ) {
 
 	this.intensity = ( intensity !== undefined ) ? intensity : 1;
 	this.distance = ( distance !== undefined ) ? distance : 0;
+	this.decayExponent = ( decayExponent !== undefined ) ? decayExponent : 1;	// for physically correct lights, should be 2.
```

Comment:
Hmm... have 2 as the default is the right thing to do.  We ran into issues with that as the default in https://Clara.io though because you now have to get the light intensity right for the scale of scene you have -- and thus any default light intensity you use with a falloff of 2 are either too bright or too dark for arbitrary scenes.  Thus we have left our decayExponent as 0 in Clara.io.

I think if you use the default of 2, a lot of people will complain of lights being too bright or too dark.  Maybe we just have to live with this?

V-Ray uses the term ```decay``` but they have made is so that 0 is no decay and 1 is physical decay (which would be 2 for decayExponent.)  We could match V-Ray exactly, thus change the name and change the scale.  Having physical at decayExponent 2 is a little non-obvious.